src/encoded/types/file.py

```diff
@@ -139,7 +139,9 @@ def __name__(self):
     def unique_keys(self, properties):
         keys = super(File, self).unique_keys(properties)
         if properties.get('status') != 'replaced':
-            if 'md5sum' in properties:
+            file_output_category = self.schema['output_type_output_category'].get(properties.get('output_type'))
+            if ((properties.get('lab') != 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d' or 
```

Comment:
It would be nice if this were wrapped in function with an informative named that returned True/False. Something like `if is_md5sum_contrained_file():` You would probably have to pass in properties and output_category etc. but would allow for future extension. It would also be nice if you defined constant at the top `CHERRY_LAB_UUID = 'a558111b-4c50-4b2e-9de8-73fd8fd3a67d'` (or whatever it is) and use the variable name instead of the uuid so it's clearer what it is.

Reply:
created external function following your suggestion